pkg/kubelet/server/server.go

```diff
@@ -244,8 +246,11 @@ func NewServer(
 		// To maintain backward compatibility serve logs only when enableDebuggingHandlers is also enabled
 		// see https://github.com/kubernetes/kubernetes/pull/87273
 		server.InstallSystemLogHandler(enableSystemLogHandler)
-		if enableContentionProfiling {
-			goruntime.SetBlockProfileRate(1)
+		if enableProfiling {
+			server.InstallProfilingHandlers()
```

Comment:
nit: Would be nice to say "profiling is disabled" when enableProfiling is false and debuggingHandler is enabled. Can you also add a unit test for this change.